app/views/ReportUserView/UserAvatarAndName.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Text, View } from 'react-native';
+
+import Avatar from '../../containers/Avatar';
+import styles from './styles';
+import { useTheme } from '../../theme';
+
+const UserAvatarAndName = ({ username, rid, name }: { username: string; rid?: string; name: string }) => {
```

Comment:
I'm not a huge fan of this pattern. I can see some edge cases where it makes sense, but this is not the case.
`UserAvatarAndName` is limiting. What if we want to add username, roles, timezone, etc?

We could pretty much call it `UserInfo`, `Header`, `UserData`, `ReportedUser`, etc.